packages/core/src/integrations/extraerrordata.ts

```diff
@@ -0,0 +1,86 @@
+import { addGlobalEventProcessor, getCurrentHub } from '@sentry/hub';
+import { Integration, SentryEvent, SentryEventHint } from '@sentry/types';
+import { isError } from '@sentry/utils/is';
+
+/**
+ * Just an Error object with arbitrary attributes attached to it.
+ */
+interface ExtendedError extends Error {
+  [key: string]: any;
```

Comment:
Not sure what your TS version is, but you might want to use `unknown`.

Reply:
Good point, we never actually used `unkown` yet because we started with typescript < 3.
But it makes sense here.